packages/core/src/contracts/tcr/civilTCR.ts

```diff
@@ -532,19 +558,19 @@ export class CivilTCR extends BaseWrapper<CivilTCRContract> {
     return this.challengeWithURI(listingAddress, uri);
   }
 
-  public async requestAppeal(listingAddres: EthAddress): Promise<TwoStepEthTransaction> {
-    return createTwoStepSimple(this.ethApi, await this.instance.requestAppeal.sendTransactionAsync(listingAddres));
+  public async requestAppeal(listingAddress: EthAddress): Promise<MultisigProxyTransaction> {
+    return this.multisigProxy.requestAppeal.sendTransactionAsync(listingAddress);
   }
 
-  public async grantAppeal(listingAddres: EthAddress): Promise<TwoStepEthTransaction> {
-    return createTwoStepSimple(this.ethApi, await this.instance.grantAppeal.sendTransactionAsync(listingAddres));
+  public async grantAppeal(listingAddress: EthAddress): Promise<MultisigProxyTransaction> {
+    return this.multisigProxy.grantAppeal.sendTransactionAsync(listingAddress);
   }
 
-  public async challengeGrantedAppeal(listingAddres: EthAddress, data: string = ""): Promise<TwoStepEthTransaction> {
-    return createTwoStepSimple(
-      this.ethApi,
-      await this.instance.challengeGrantedAppeal.sendTransactionAsync(listingAddres, data),
-    );
+  public async challengeGrantedAppeal(
+    listingAddress: EthAddress,
+    data: string = "",
```

Comment:
This line is only changed because the return type changed to longer `MultisigProxyTransaction` so prettier broke params onto separate lines. Not sure what this will be in the end so I'll leave it for now.